admin/picture_modify.php

```diff
@@ -284,7 +284,7 @@
   'file' => l10n('Original file : %s', $row['file']),
   'add_date' => l10n('Posted %s on %s', time_since($row['date_available'], 'year'), format_date($row['date_available'], array('day', 'month', 'year'))),
   'added_by' => l10n('Added by %s', $row['added_by']),
-  'size' => $row['width'].'&times;'.$row['height'].' pixels, '.sprintf('%.2f', $row['filesize']/1024).'MB',
+  'size' => l10n('%s pixels, %.2f MB', $row['width'].'&times;'.$row['height'], $row['filesize']/1024),
```

Comment:
The `%s pixels, %.2f MB` message should also be added to `language/en_UK/admin.lang.php`, I think.